hedera-node/hedera-file-service-impl/src/main/java/com/hedera/node/app/service/file/impl/FileStore.java

```diff
@@ -25,6 +25,7 @@
 /**
  * Base class for {@link ReadableFileStoreImpl} and {@link WritableFileStore}.
  */
+@SuppressWarnings("java:S1118") // suppressing the warning to have a private constructor. This is not utility class
```

Comment:
Instead of suppressing this, would the warning go away with a default protected constructor?